services/consensusalgo/leanhelixconsensus/test/leader_elected_by_sync_does_not_immediately_propose_block_test.go

```diff
@@ -24,44 +21,39 @@ import (
 // This causes it to get stuck on GetTransactionsForOrdering (9 seconds when no traffic)
 // and broadcast large pre prepares that nobody cares about to everybody (network pollution).
 
-func TestService_SlowBlockCreationDoesNotHurtBlockSync(t *testing.T) {
+func TestService_ShyLeader_LeaderElectedByBlockSyncDoesNotProposeABlockOnFirstView(t *testing.T) {
 	test.WithContext(func(ctx context.Context) {
 		h := newLeanHelixServiceHarness(0).start(t, ctx)
 
-		isLeanHelixStuckOnCreatingABlock := false
+		h.expectConsensusContextRequestNewBlockNotCalled()
 
-		h.expectConsensusContextRequestOrderingCommittee(0) // we're index 0 (first time called)
-		h.expectConsensusContextRequestOrderingCommittee(0) // we're index 0 (second time called)
-		// TODO REMOVE LINE h.expectGossipSendLeanHelixMessage()
-		h.consensusContextRespondWithVerySlowBlockCreation(&isLeanHelixStuckOnCreatingABlock)
+		h.expectConsensusContextRequestOrderingCommittee(0, 1) // we're index 0 (first time called)
+		mockFuncRequestOrderingCommittee := h.consensusContext.Functions[len(h.consensusContext.Functions)-1]
```

Comment:
this is fragile if I understand what you did here

Reply:
why?

Reply:
because if it's no longer the last function?

Reply:
its gone already... this was garbage code i forgot to clean up.